loom/graphics/gfxGraphics.cpp

```diff
@@ -286,6 +287,10 @@ int Graphics::render(lua_State *L)
     lmscalar prevAlpha = object->alpha;
     object->alpha = prevAlpha*alpha;
 
+    // Reset state
+    Graphics_SetCurrentGLState(GFX_OPENGL_STATE_QUAD);
+    Graphics_InvalidateGLState(GFX_OPENGL_STATE_QUAD);
```

Comment:
Based on the implementation that resets all GL state, we should probably have something explicit for that, but those two lines do for now.